ts/features/payments/transaction/components/WalletTransactionDetailsList.tsx

```diff
@@ -46,18 +47,23 @@ export const WalletTransactionDetailsList = ({
     IUV: pipe(getTransactionIUV(transaction.description), O.toUndefined)
   };
 
+  const amountText = formatNumberCentsToAmount(
+    transaction.amount.amount,
+    true,
+    "right"
+  );
+
   return (
     <>
       <ListItemTransaction
         title={cleanTransactionDescription(transaction.description)}
         subtitle={transaction.merchant}
-        transactionStatus="success"
-        transactionAmount={formatNumberCentsToAmount(
-          transaction.amount.amount,
-          true,
-          "right"
-        )}
-        hasChevronRight
+        transaction={{
+          amount: amountText,
+          amountAccessibilityLabel:
+            getAccessibleAmountText(amountText) ?? amountText
```

Comment:
Since `amountString` returns an empty string as its fallback in the worst-case scenario, is the additional fallback operation truly necessary?

Reply:
Same as above, the `getAccessibleAmountText` could return an `undefined` value, it means that the fallback operation is necessary in this case.